src/Compilers/CSharp/Portable/Parser/DocumentationCommentParser.cs

```diff
@@ -1262,8 +1262,23 @@ private CrefParameterSyntax ParseCrefParameter()
                     break;
             }
 
+            SyntaxToken readOnlyOpt = null;
+            if (CurrentToken.Kind == SyntaxKind.ReadOnlyKeyword && refKindOpt is not null)
+            {
+                if (refKindOpt.Kind != SyntaxKind.RefKeyword)
+                {
+                    // if we encounter `readonly` after `in` or `out`, we place the `readonly` as skipped trivia on the previous keyword
+                    var misplacedToken = AddErrorAsWarning(EatToken(), ErrorCode.ERR_RefReadOnlyWrongOrdering);
```

Comment:
I didn't follow why this case is just a warning. Should this be an error? 
Other places in doc-comment parser that use `AddTrailingSkippedSyntax` report an error.

Reply:
Never mind, Aleksey clarified offline. The doc comments never produce errors